lib/workload/stateless/metadata_manager/src/metadata_manager/urls/base.py

```diff
@@ -0,0 +1,12 @@
+from django.urls import path, include
+
+from metadata_manager.routers import OptionalSlashDefaultRouter
+from metadata_manager.viewsets.metadata import MetadataViewSet
+
+router = OptionalSlashDefaultRouter()
+router.register(r"metadata", MetadataViewSet, basename="metadata")
+
+urlpatterns = [path("iam/", include(router.urls)), path("", include(router.urls))]
```

Comment:
Do we need the "iam/" path?

Reply:
Yup, can omit `iam/` for now. Will; could you also update in skel profiles laster, pls.